pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -474,23 +474,6 @@ var testGrid = []testCase{
 		analyzer:   &deployment.ServiceAssociationAnalyzer{},
 		expected:   []message{},
 	},
-	{
-		name: "regexes",
-		inputFiles: []string{
-			"testdata/virtualservice_regexes.yaml",
-		},
-		analyzer: &virtualservice.RegexAnalyzer{},
-		expected: []message{
-			{msg.InvalidRegexp, "VirtualService bad-match"},
```

Comment:
The message is not used anywhere anymore?

Reply:
removed